content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/hierarchy.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Network Hierarchy Entities
+description: A reference for creating hierarchical network entities.
+linktitle: Hierarchies
+---
+
+*Network Hierarchies* provide a convenient way to group related entities under one logical group. Entities in such a hierarchy process their network input together as one, including their input creation, processing and rollback.
+
+{{<important>}}
+Network Hierarchies are designed for small groups of entities.
+
+By default the limit is set to 16 entities. You can modify this limit with CVar: `bg_hierarchyEntityMaxLimit`.
+{{</important>}}
+
+## Hierarchical Components
+
+In order to declare a network hierarchy, you have to use a combination of hierarchical components: `NetworkHierarchyRootComponent` and `NetworkHierarchyChildComponent`. `NetworkHierarchyRootComponent` marks an entity as the root of a hierarchy and it should be placed on the highest parent of your entity prefab or a group of entities. `NetworkHierarchyRootComponent` marks an entity as a child of a hierarchy. It should be added on child entities under the entity with `NetworkHierarchyRootComponent`.
+
+Network hierarchies can be created either in the Editor or by parenting entities at runtime.
+
+### Creating Network Hierarchies in the Editor
+
+1. Creating a network entity with all its network common components:
+    - Network Binding Component
+    - Network Transform Component
+
+
+1. Add a Mesh component for visual information in the Editor.
+
+1. Here is an example of a simple network entity:
+
+    ![Starting Network Entity](/images/user-guide/gems/reference/multiplayer/starting_network_entity.png)
+
+1. In order to make it a hierarchical root entity, add a **Network Hierarchy Root** component:
+
+    ![Starting Hierarchy Root Entity](/images/user-guide/gems/reference/multiplayer/starting_hierarchy_root_entity.png)
+
+This entity will be a parent entity for hierarchical child entities.
+
+5. A hierarchical child entity is formed by adding a **Network Hierarchy Child** component.
+
+    ![Starting Hierarchy Child Entity](/images/user-guide/gems/reference/multiplayer/starting_hierarchy_child_entity.png)
+
+Notice that this entity is a child entity to the root entity.
```

Comment:
```suggestion
1. In order to make it a hierarchical root entity, add a **Network Hierarchy Root** component:

    ![Starting Hierarchy Root Entity](/images/user-guide/gems/reference/multiplayer/starting_hierarchy_root_entity.png)

    This entity will be a parent entity for hierarchical child entities.

1. A hierarchical child entity is formed by adding a **Network Hierarchy Child** component.

    ![Starting Hierarchy Child Entity](/images/user-guide/gems/reference/multiplayer/starting_hierarchy_child_entity.png)

    Notice that this entity is a child entity to the root entity.
```

Markdown indentation for lists is extremely whitespace-sensitive. See [Markdown Cheatsheet](https://www.markdownguide.org/basic-syntax/#ordered-lists). If you have to manually label a step in a list, it means that you tried to place text inline with a step and used the wrong amount of whitespace.

Reply:
Corrected whitespace, thanks for the doc link!